realm/src/androidTest/java/io/realm/DynamicRealmObjectTest.java

```diff
@@ -232,7 +232,7 @@ public void testSetXXXNullValues() {
                     case BINARY:
                     case DATE:
                     default:
-                        continue; // Ignore other types for now
+                        continue; // Ignore other types for now until null support is merged.
```

Comment:
This is fixed in https://github.com/realm/realm-java/pull/1548 instead. This PR does not contain null support.